File: letter/ru/Н.py
from common import *

expr = Eq(
    div(
        (Abs(x) - S(1) / 2) * (y - eps) * sqrt(S(1) / 2 + eps - Abs(x)),
        sqrt(1 - Abs(y))
    )
    ,
    0
)
size = 1

File: letter/ru/Н.js
import { Letter } from '../../Letter.js';

const expr = '(abs(x) - 1/2) * (y - 1/10) * sqrt(1/2 + 1/10 - abs(x)) / sqrt(1 - abs(y))';

export const letter = new Letter(expr, 1);
